### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kuraga**
More concrete instructions about setting the superuser account credentials.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,40 +26,49 @@ You can try the [annotation demo](http://doccano.herokuapp.com).
 
 ## Usage
 
-Two options to run doccano:
+Two options to run doccano: production or development.
```
Comment (Hironsan) [inline README.md:29]: The list is preferable:

- Production
- Development

Comment (kuraga) [reply README.md]: Done.

File: README.md [hunk 2]
```diff
@@ -28,38 +28,50 @@ You can try the [annotation demo](http://doccano.herokuapp.com).
 
 Two options to run doccano:
 
-- Production
-- Development
+- production,
+- development.
 
-In any case, you need Docker and Docker Compose. Anyway, you need to clone the repository:
+To use Doccano, please follow:
```
Comment (icoxfog417) [inline README.md:34]: Doccano => doccano

Comment (kuraga) [reply README.md]: Oh, sorry, forgot. Done.

**Comment by kuraga** [comment]
Thanks!

**Pull request merged**